loom/script/runtime/lsLuaState.cpp

```diff
@@ -526,6 +509,30 @@ void LSLuaState::finalizeAssemblyLoad(Assembly *assembly, utArray<Type *>& types
     assembly->bootstrap();
 }
 
+void LSLuaState::cleanUpMissingTypes()
```

Comment:
This was done before in `finalizeAssemblyLoad`, but because of cross-assembly dependencies, this is done after loading everything.